docs/pages/push-notifications/using-fcm.mdx

```diff
@@ -3,7 +3,7 @@ title: Using FCM for Push Notifications
 sidebar_title: Using FCM
 ---
 
-**Firebase Cloud Messaging is required for all Android apps using Expo SDK**, unless you're still running your app in the Expo Go app. To set up your Expo Android app to get push notifications using your own FCM credentials, follow this guide closely.
+**Firebase Cloud Messaging is required for all Android apps using Expo SDK**, unless you're still trying your project in the Expo Go app. To set up your Expo Android app to get push notifications using your own FCM credentials, follow this guide closely.
```

Comment:
@EvanBacon, do you think, instead of the word running, we can rely on "testing" as it makes more context or am thinking about this wrong?